x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/setup_technology.ts

```diff
@@ -74,46 +87,75 @@ export function useSetupTechnology({
   setSelectedPolicyTab: (tab: SelectedPolicyTab) => void;
   packageInfo?: PackageInfo;
 }) {
-  const { isAgentlessEnabled, isAgentlessIntegration } = useAgentless();
+  const { cloud } = useStartServices();
+  const {
+    isAgentlessEnabled,
+    isAgentlessIntegration,
+    isAgentlessCloudEnabled,
+    isAgentlessServerlessEnabled,
+  } = useAgentless();
```

Comment:
nice! 🚀

Reply:
Ya,  it flows better now.  Thanks for the 👁️